src/python_service_template/infrastructure/health.py

```
@@ -0,0 +1,71 @@
+class HealthChecker(abc.ABC, t.Generic[T]):
+    def __init__(self, coffee_client: CoffeeClient, app_version: str, git_sha: str) -> None:
+        self.coffee_client = coffee_client
+        self.app_version = app_version
+        self.git_sha = git_sha
+
+    async def check(self) -> T:
+        checks: dict[str, HealthIndicator] = {}
+
+        try:
+            healthy = await self.coffee_client.healthy()
+        except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)